tree/dataframe/src/RDataFrame.cxx

```diff
@@ -481,6 +481,20 @@ sum = df.Filter("myFilter(x)").Sum("y")
 print(sum.GetValue())
 ~~~
 
+Alternatively, you can also pass the full RDataFrame object to C++ using the ROOT::RDF::AsRNode helper in Python, which casts any RDataFrame node to ROOT::RDF::RNode:
+
+~~~{.python}
+ROOT.gInterpreter.Declare('''
+ROOT::RDF::RNode MyTransformation(ROOT::RDF::RNode& df) {
```

Comment:
I am not sure taking by non-const reference would compile as we are passing in a temporary. But passing by value is totally ok (we also return by value so it's also good to be consistent I guess): 

```suggestion
ROOT::RDF::RNode MyTransformation(ROOT::RDF::RNode df) {
```